docs/source/releases/v1_13_0a0.rst

```diff
@@ -16,6 +16,9 @@ Version 1.13.0a0 (2024-04-24)
 * Enhancements
 
   * Initial Stage of the GeoIPS CLI
+
+* Documentation
+
```

Comment:
It looks like you forgot to add a bullet here

Reply:
Thank you for adding the section to the release notes!

Reply:
I think this is resolved?